### DISCUSSION TITLE
关于在termux上部署Pandora ai

### DISCUSSION THREAD

**Discussion by xuyufenfei** [viopsa233/AI-testers-Forum#84] [Q&A]
node gpt已经搞定了，但下载下来的文件没有.env，无论是git clone还是下载下来软链接，希望有大佬提供解决思路，或者探索出部署方法

**Comment by viopsa233** [comment]
为啥还在用Pandora ai?

**Reply by xuyufenfei** [reply]
那大佬有什么好用的第三方客户端推荐一下吗

**Reply by viopsa233** [reply]
> 那大佬有什么好用的第三方客户端推荐一下吗

chatsydney，有人做了一个现成的

**Reply by xuyufenfei** [reply]
可是chat Sydney发不了图

**Comment by viopsa233** [comment]
> 可是chat Sydney发不了图

有人魔改了一下,现在支持画图还有图像识别了
[ChatSydney](https://github.com/SoraRoseous/ChatSydney-react)

**Reply by xuyufenfei** [reply]
大佬给个链接

**Reply by viopsa233** [reply]
https://github.com/SoraRoseous/ChatSydney-react

**Reply by codescipio** [reply]
这个好像还是会报错，我挂在replit上一直弹
{"stack":"Error: Failed to construct 'WebSocket': An insecure WebSocket connection may not be initiated from a page loaded over HTTPS.\n    at <anonymous>:460:25\n    at new Promise (<anonymous>)\n    at <anonymous>:459:18\n    at Generator.next (<anonymous>)\n    at asyncGeneratorStep (<anonymous>:1:103)\n    at _next (<anonymous>:2:194)\n    at <anonymous>:2:364\n    at new Promise (<anonymous>)\n    at <anonymous>:2:97\n    at _connectWebSocket (<anonymous>:469:34)\n    at connectWebSocket (<anonymous>:455:34)\n    at <anonymous>:471:13\n    at Generator.next (<anonymous>)\n    at asyncGeneratorStep (<anonymous>:1:103)\n    at _next (<anonymous>:2:194)\n    at <anonymous>:2:364\n    at new Promise (<anonymous>)\n    at <anonymous>:2:97\n    at streamOutput (<anonymous>:607:20)\n    at <anonymous>:434:30\n    at Generator.next (<anonymous>)\n    at asyncGeneratorStep (<anonymous>:1:103)\n    at _next (<anonymous>:2:194)\n    at <anonymous>:2:364\n    at new Promise (<anonymous>)\n    at <anonymous>:2:97\n    at sendMessage (<anonymous>:449:20)\n    at handleUserInputKeyDown (<anonymous>:614:7)\n    at Object.Ts (https://cdn.skypack.dev/-/react-dom@v17.0.1-oZ1BXZ5opQ1DbTh7nu9r/dist=es2019,mode=imports,min/optimized/react-dom.js:5:9340)\n    at zs (https://cdn.skypack.dev/-/react-dom@v17.0.1-oZ1BXZ5opQ1DbTh7nu9r/dist=es2019,mode=imports,min/optimized/react-dom.js:5:9494)\n    at Ms (https://cdn.skypack.dev/-/react-dom@v17.0.1-oZ1BXZ5opQ1DbTh7nu9r/dist=es2019,mode=imports,min/optimized/react-dom.js:5:9551)\n    at Vo (https://cdn.skypack.dev/-/react-dom@v17.0.1-oZ1BXZ5opQ1DbTh7nu9r/...
